typeclasses/characters.py

```diff
@@ -91,6 +75,71 @@ def str(self, amount):
 
         self.db.slots = {}
 
+    # helper method, checks if stat is valid
+    def find_stat(self, stat):
+        if stat in self.db.primary_traits:
+            return self.db.primary_traits[stat]
+        elif stat in self.db.secondary_traits:
+            return self.db.secondary_traits[stat]
+        else:
+            return None
+
+    # helper method, checks if race gets extra language points
+    def determine_language_points(self):
+        if self.bonuses['languages']:
+            return self.bonuses['languages']
+
+    def base_stat(self, stat, amount):
+        # sets the secondary traits
+        if stat == 'vitality':
+            base_stat('health', amount)
+            base_stat('stamina', amount)
+            base_stat('fortitude', amount)
+            base_stat('reflex', amount)
+
+        if stat == 'intelligence':
+            bonus_language_points = determine_language_points()
+            base_stat('languages', amount + bonus_language_points)
+            base_stat('will', amount)
+              
+        # as per the OA blue rulebook mana can never exceed 10 (page 49)
+        if stat = 'mana':
+            if amount > 10:
+                self.db.secondary_traits['mana'].base = 10
+                return
+            else:
+                self.db.secondary_traits['mana'].base = amount
+                return
+
+        stat_valid = find_stat(stat)
```

Comment:
`stat_valid` is a name that sounds like it would be either `True` or `False`. As it is, it is a stat object. `stat` is taken, however, so I wonder if this could be something like `valid_stat` instead.